jpyutil.py

```diff
@@ -38,8 +38,18 @@
 __version__ = "0.10.0.dev1"
 
 
-# Uncomment for debugging
-# logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.DEBUG)
+# Setup a dedicated logger for jpyutil.
+# This way importing jpyutil does not interfere with logging in other modules
+
+logger = logging.getLogger('jpyutil')
+# set to logging.DEBUG for debugging
+logger.setLevel(logging.INFO)
+
+ch = logging.StreamHandler()
```

Comment:
Instead of `ch` use `sh`.

Reply:
I copied the code pretty much in verbatim from the Python documentation https://docs.python.org/3.6/howto/logging.html#configuring-logging, where they use **ch** as well, for console handler. But I've no problem changing it to **sh**. I just tried to stay consistent with the Python docs.